### PR TITLE
Fixes #250

### LINKED ISSUE #250: Add process propagation qualities
Some processes propagate spatially. Examples include and an action potential propagating along an axon - or spreading through the heart (cardiac conduction); the propagation of a wave of cAMP across a population of Dictyostelium; the propagation of a wave of differentiation (the  morphogenetic furrow) across a developing eye discs; a peristaltic contraction wave propagates down the oesophagus, transporting food.

We can't use PATO: velocity to record the speed of propagation as this is a physical object property.  We therefore need new properties for phenotypes that effect the velocity of propagation of a process.

### PR DESCRIPTION
**Pull request by dosumis**

### CODE REVIEW

**Comment by ybradford** [comment]
@dosumis  this looks good to me

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -19822,6 +19822,33 @@ is_a: PATO:0000141 ! structure
...
+name: propagation velocity
+def: "The velocity of spatial propagation of a process. Examples include the velocity of propagation of an action potential along an axon." [orcid:0000-0002-7073-9172]
+is_a: PATO:0002062 ! physical quality of a process
+property_value: created_by "dos" xsd:string
+creation_date: 2019-07-18T18:24:15Z
+
+[Term]
+id: PATO:0010003
+name: increased propagation velocity
+def: "A propagation velocity that is higher than normal." [] {http://www.w3.org/2002/07/owl#backwardCompatibleWith="orcid:0000-0002-7073-9172"}
```
Comment (matentzn) [inline src/ontology/pato-edit.obo]: Orcid notation seems off. Moreover, shouldnt orcids be always full IRIs, rather than short forms?

Comment (dosumis) [reply src/ontology/pato-edit.obo]: Fixed

**Comment by dosumis** [comment]
@ybradford - could you approve as a reviewer - rather than just in comments?  This is needed for me to merge pull?  Many thanks, David

**Comment by dosumis** [comment]
> @ybradford - could you approve as a reviewer - rather than just in comments? This is needed for me to merge pull? Many thanks, David

Scratch that.  Not needed this time.  I've just realised I can over-ride.

**Pull request merged**